draft-ietf-mls-protocol.md

```diff
@@ -1629,24 +1628,24 @@ struct {
     opaque authenticated_data<V>;
 
     ContentType content_type;
-    select (MLSContent.content_type) {
+    select (GroupContent.content_type) {
```

Comment:
I suggested "MessageContent", "MessageContentTBS", etc. as opposed to using the "Group-" prefix. It got some 👍🏻, but no other reaction. Do you still prefer the "Group-" prefix?